tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain.md

```diff
@@ -0,0 +1,108 @@
+---
+title: From Relay Chain to Parachain
+description: Learn how to perform a reserve-backed asset transfer between a relay chain and a parachain using XCM for cross-chain interoperability.
+---
+
+# From Relay Chain to Parachain
+
+## Introduction
+
+[Cross-Consensus Messaging (XCM)](/develop/interoperability/intro-to-xcm/){target=\_blank} facilitates asset transfers both within the same consensus system and between different ones, such as between a relay chain and its parachains. For cross-system transfers, two main methods are available:
+
+- **Asset teleportation** - a simple and efficient method involving only the source and destination chains, ideal for systems with a high level of trust
+- **Reserve-Backed transfers** - involves a trusted reserve holding real assets and mints derivative tokens to track ownership. This method is suited for systems with lower trust levels
+
+In this tutorial, you will learn how to perform a reserve-backed transfer between a relay chain (Polkadot) and a parachain (Astar).
+
+## Prerequisites
+
+To send messages between different consensus systems, you must first open HRMP channels. For detailed guidance, refer to the [XCM Channels](/develop/interoperability/xcm-channels/#xcm-channels){target=\_blank} article before for further information about.
+
+This tutorial uses Chopsticks to fork a relay chain and a parachain connected via HRMP channels. For more details on this setup, see the [XCM Testing](/tutorials/polkadot-sdk/testing/fork-live-chains/#xcm-testing){target=\_blank} section on the Chopsticks page.
+
+## Setup
+
+To simulate XCM operations between different consensus systems, start by forking the network with the following command:
+
+```bash
+chopsticks xcm -r polkadot -p astar
+```
+After executing this command, the relay chain and parachain will expose the following WebSocket endpoints:
+
+=== "Polkadot (relay chain)"
+
+    ```bash
+    ws://localhost:8001
+    ```
+
+=== "Astar (parachain)"
+
+    ```bash
+    ws://localhost:8000
+    ```
+
+You can connect these endpoints using the [Polkadot.Js Apps](https://polkadot.js.org/apps/){target=\_blank} interface. Open two browser tabs and add the custom endpoint for each chain as shown below:
+
+![](/images/tutorials/interoperability/xcm-transfers/xcm-transfers-01.webp)
+
+Click **Switch** to connect to the respective network.
+
+## Procedure to Execute a Reserve-Backed Transfer
+
+Using their reserve system, this method facilitates asset transfers from a local chain to a destination chain. Fees on the destination chain are deducted from the asset specified in the assets vector at the `fee_asset_item` index, covering up to the specified `weight_limit.` The operation fails if the required weight exceeds this limit, potentially putting the transferred assets at risk.
```

Comment:
`Using their reserve system, this method facilitates asset transfers from a local chain to a destination chain.` 
 This first sentence is weird, does not seem accurate